sfputil/main.py

```diff
@@ -329,14 +331,16 @@ def get_path_to_port_config_file():
     hwsku_path = "/".join([platform_path, hwsku])
 
     # First check for the presence of the new 'port_config.ini' file
-    port_config_file_path = "/".join([hwsku_path, "port_config.ini"])
+    port_config_file_path = "/".join([platform_path, PLATFORM_JSON])
+    if not os.path.isfile(port_config_file_path):
+        # platform.json doesn't exist. Try loading the legacy 'port_config.ini' file
+        port_config_file_path = "/".join([hwsku_path, PORT_CONFIG_INI])
     if not os.path.isfile(port_config_file_path):
         # port_config.ini doesn't exist. Try loading the legacy 'portmap.ini' file
-        port_config_file_path = "/".join([hwsku_path, "portmap.ini"])
+        port_config_file_path = "/".join([hwsku_path, PORTMAP_INI])
```

Comment:
`portmap.ini` is legacy, and was deprecated years ago in favor of `port_config.ini`. I think at this point, we can remove reference to this file entirely.

@lguohan: Do you agree?

Reply:
@samaity: Please remove the `portmap.ini` references as part of this PR.

Reply:
#done